test/test_issue1387.m

```diff
@@ -0,0 +1,40 @@
+function test_issue1387
+
+%datadir = dccnpath('/home/common/matlab/fieldtrip/data/');
```

Comment:
I have downloaded the test file and copied it to the compute cluster

```
roboos@mentat002> mkdir /home/common/matlab/fieldtrip/data/test/issue1387
roboos@mentat002> mv Test_4.edf /home/common/matlab/fieldtrip/data/test/issue1387
roboos@mentat002> ls /home/common/matlab/fieldtrip/data/test/issue1387
Test_4.edf
```

if you use
```
fileName = dccnpath('/home/common/matlab/fieldtrip/data/test/issue1387/Test_4.edf');
```
then you should see on your local computer that `dccnpath` returns the local file (assuming it is in your present working directory. So you can run the test script locally.

On the DCCN compute environment `dccnpath` either returns the original name, or the windows version of the name with `h:` instead of `/home`, and with backward slashes.

Reply:
OK, path added.